content/en/_index.html

```diff
@@ -21,18 +21,18 @@
 
 {{< blocks/lead color="primary" >}}
 XLT is a load and performance test tool that differs from other approaches due to its focus
-on high quality reporting and the Java-based programming approach. There are many ways to work with XLT and this documentation will show you some. This documentation will explain features and functionalities and also wants to inspire you to take your performance testing to the next level.
+on high quality reporting and the Java as programming language for the scripts. There are many ways to work with XLT and this documentation will show you some. It will explain features and functionalities and hopefully inspires you to take your performance testing to the next level.
 {{< /blocks/lead >}}
 
 {{< blocks/section color="dark" type="features" >}}
 
-{{% blocks/feature icon="fa-lightbulb" title="Open Source" url="/docs/examples/" %}}
-XLT is licensed under Apache License 2.0. This give you the assurenace that the tool is here to stay and also enables you to extend it if needed.
+{{% blocks/feature icon="fa-lightbulb" title="Open Source" url="https://github.com/Xceptance/XLT/" %}}
```

Comment:
This would be another place where we could put a link to the actual content of the license. I think it should be somewhere, at least - any suggestions?

Reply:
Updated here. This is a public license text hence we link to apache